addons/account/models/account_journal_dashboard.py

```diff
@@ -348,7 +348,7 @@ def open_action(self):
             'type': invoice_type
         })
 
-        [action] = self.env.ref('account.%s' % action_name).read()
+        [action] = self.env.ref('account.%s' % action_name).with_context(ctx).read()
```

Comment:
@sja-odoo can you explain why?

Reply:
Yes @qdp-odoo 
When user traverse through vendor bills from dashboard and if there isn't any bills created yet at that time empty list help message should be formed with current journal's alias only, that is, context key `default_journal_id` need to be passed while calling `read()` of vendor bill's action and it is checked here https://github.com/odoo/odoo/pull/22158/files#diff-f3712147f4b25231caa371bd83265dbeR671